docs/versionPolicy.md

```diff
@@ -0,0 +1,212 @@
+# Versioning
+
+Running `kustomize` means one is running a
+particular version of a program, reading a
+particular version of a [kustomization] file.
+
+## Program versioning
+
+The command `kustomize version` prints a three
+field version tag (e.g. `1.0.11`) that aspires to
+[semantic versioning].
+
+When enough changes have accumulated to
+(subjectively) warrant a new release,
+a [release process] is followed, and the
+fields in the version number are bumped
+per semver.
+
+## Kustomization file versioning
+
+At the time of writing:
+
+ - A [kustomization] file is just a YAML file that
+   can be successfully parsed into a particular Go
+   struct defined in the `kustomize` binary.
+
+ - This struct does not have a version number,
+   which is the same as saying that its version
+   number matches the program's version number,
+   since it's compiled in.
+
+### Kustomization Field Change Policy
+
+- A field's meaning cannot be changed.
+
+- A field may be deprecated, then removed.
+
+- Deprecation means triggering a _minor_ (semver)
+  version bump in the program, and
+  defining a migration path in a non-fatal
+  error message.
+
+- Removal means triggering a _major_ (semver)
+  version bump, and fatal error if field encountered
+  (as with any unknown field).
+
+### The `edit fix` command
+
+This `kustomize` command reads a Kustomization
+file, translates _deprecated_ fields to new
+fields, and writes it out again in the latest
+format.
+
+This is a type version upgrade mechanism.  There
+is no downgrade capability, as there's no use case
```

Comment:
`edit fix` command doesn't provide ability to go back.
But if the users use GitOps in their workflow, they can always go back.
We might write it down if you think it's useful.

Reply:
updated, thanks!